### PR TITLE
0.67.0 release notes

### PR DESCRIPTION
**Pull request by fsamuel-bs**
**Goals (and why)**: 0.67.0 release notes

---
This change is [Reviewable](https://reviewable.io/reviews/palantir/atlasdb/2685)

### CODE REVIEW

File: docs/source/release_notes/release-notes.rst [hunk 1]
```diff
@@ -84,33 +91,103 @@ develop
...
+              - com.palantir.atlasdb.keyvalue.cassandra.QosMetrics.bytesRead
+              - com.palantir.atlasdb.keyvalue.cassandra.QosMetrics.bytesWritten
+
+           (`Pull Request <https://github.com/palantir/atlasdb/pull/2679>`__)
+
+    *    - |new| |metrics|
+         - Added metrics for cells read.
+           The read cells can be post-filtered at the CassandraKVS layer, when its value is not the latest. In this case, we have the metric:
+
+              - com.palantir.atlasdb.AtlasDbMetricNames.CellFilterMetrics.notLatestVisibleValueCellFilterCount
```
Comment (hsaraogi) [inline docs/source/release_notes/release-notes.rst]: The names will be fully qualified class names of `ValueExtractor/TsExtractor/Row(something)Extractor`. `notLatestVisibleValueCellFilterCount`? how did you infer these would be the names :( ?

File: docs/source/release_notes/release-notes.rst [hunk 2]
```diff
@@ -84,33 +91,103 @@ develop
...
+
+           The cells returned from the KVS layer are then recorded at the metric:
+
+              - com.palantir.atlasdb.AtlasDbMetricNames.numCellsRead
```
Comment (hsaraogi) [inline docs/source/release_notes/release-notes.rst]: I think this one comes from SnapshotTransaction.. etc and so on for all the metric names.

File: docs/source/release_notes/release-notes.rst [hunk 3]
```diff
@@ -84,33 +91,103 @@ develop
...
+              - com.palantir.atlasdb.AtlasDbMetricNames.numCellsReturnedAfterFiltering
+
+           (`Pull Request <https://github.com/palantir/atlasdb/pull/2671>`__)
+
+
+    *    - |new| |metrics|
+         - Added metrics for written cells.
+           Empty cells are filtered out at the transaction layer, where we record the metric:
+
+              - com.palantir.atlasdb.AtlasDbMetricNames.CellFilterMetrics.emptyValuesCellFilterCount
```
Comment (hsaraogi) [inline docs/source/release_notes/release-notes.rst]: These are for cells read.

File: docs/source/release_notes/release-notes.rst [hunk 4]
```diff
@@ -84,33 +91,103 @@ develop
...
+
+           The cells actually written to Cassandra are then recorded at the metric:
+
+              - com.palantir.atlasdb.AtlasDbMetricNames.bytesWritten
+
+           (`Pull Request <https://github.com/palantir/atlasdb/pull/2671>`__)
+
+    *    - |new| |metrics|
+         - A metric was added for the cases where a large read was made. Before we to logged a warning. The metric is:
```
Comment (tboam) [inline docs/source/release_notes/release-notes.rst]: "Before we to logged a warning" doesn't make sense

File: docs/source/release_notes/release-notes.rst [hunk 5]
```diff
@@ -84,33 +91,103 @@ develop
...
+
+              - com.palantir.atlasdb.AtlasDbMetricNames.CellFilterMetrics.notLatestVisibleValueCellFilterCount
+
+           The cells returned from the KVS layer are then recorded at the metric:
+
+              - com.palantir.atlasdb.AtlasDbMetricNames.numCellsRead
+
+           Such cells can also be filtered out at the transaction layer, due to the Transaction Protocol. There, we have the metrics which record the filtered out cells:
+
+              - com.palantir.atlasdb.AtlasDbMetricNames.CellFilterMetrics.commitTsGreaterThatTxTsCellFilterCount
```
Comment (hsaraogi) [inline docs/source/release_notes/release-notes.rst]: Lets format the names too!

**Pull request merged**